### ISSUE TITLE
Placement of Point cloud in scene

### ISSUE THREAD

**Issue by bjornsyse** [unitycoder/UnityPointCloudViewer#127]
Hi

It seems the rendering of my pointcloud is not honoring the transform values, and I can't control where in the scene it's renderered. Is this expected behaviour? 

![image](https://user-images.githubusercontent.com/26255606/149555630-9eb71cc2-50e0-4c51-a3aa-d539f4b3d7dc.png)

**Comment by unitycoder** [comment]
yes thats the default way (for v1/v2 (.bin, .ucpc) viewer there are shaders with offset variable to translate cloud,
and in next version there is shader that enables transform follow (pos, rot, scale), but for v1/v2 formats.

V3 tiles format is bit different, as its tile based, cannot move points, unless update all tile culling positions also..
i can check if that gets complicated/slow.. ill reply here.

current "workaround" could be adding manual -offset to cloud during conversion.. but of course not very useful.

**Comment by Rickmc3280** [comment]
another workaround could be a script that changes all of those variables in the tiles and the root file. Seems like it would be pretty easy since the root file contains the locations of all files yes?

**Comment by unitycoder** [comment]
yes, that would be possible (maybe faster than converting again, at least for large clouds)..
but then again, iterating hundreds of files wouldnt be too fast either.. (at least if need to move transform many times)

so at that point, easier to add method that offsets loaded point data, re-inits cloud.

v3 data formats are described here, its just raw point data:
https://github.com/unitycoder/UnityPointCloudViewer/wiki/Binary-File-Format-Structure#custom-v3-tiles-pcroot-and-pct-rgb

**Comment by Rickmc3280** [comment]
Well If you ever make a binary version Can it look like this? xD  It would make it very easy to manipulate the data if the files were in binary vs ascii.  the conversion takes more time than its worth.  Would be easier just to create a viewer for people who want to read the files. 
![image](https://user-images.githubusercontent.com/47441615/149584700-0f1e5357-2cab-4450-804a-4437279e4f15.png)

**Comment by bjornsyse** [comment]
Ah ok. I was not aware at all. It seems this makes it quite not so usefull at all for my purposes then. But you're saying there are offset values in the V2 version. Perhaps I'll go that route for now, then but I'd have to completely decimate my pointcloud in that case. Or perhaps re-convert LAS-file with offset values, that could possibly work aswell. Thanks for info! Hoping for the next version then.

**Comment by bjornsyse** [comment]
On a side note, is this something that could be of interest for this plugin? 

https://docs.unity3d.com/Packages/com.unity.visualeffectgraph@12.0/manual/whats-new-12.html

![image](https://user-images.githubusercontent.com/26255606/149735332-1ecccd47-8752-4ab9-9870-5a2f77c43eb5.png)

**Comment by unitycoder** [comment]
if you send me an email, i can attach the v2 translation shader version for testing.

for v3 tiles:
Is it enough to move it during Start() to match transform pos, or you'd like to move it dynamically during playing also?

VFX, noticed that feature but haven't tested yet.. i'll put it big higher on the test list, to see if its useful. (not sure how it would play with tiles version & culling system though. but for single cloud(s) should certainly work)

**Comment by bjornsyse** [comment]
Thanks! I sent you an email.

about the v3 tiles question, I really think for most cases Start() adjustment would be sufficient. At least for the use cases I imagine for my own projects.

**Comment by unitycoder** [comment]
made initial tests for this issue,
works if manually loop & offset loaded points, but that would be slow..
so it needs custom shader that offsets points too.

this should work, so i expect it to come in the next update.

**Comment by unitycoder** [comment]
ok, it works for the basic v3 viewer, not packed version yet..
![image](https://github.com/unitycoder/UnityPointCloudViewer/assets/5438317/42c4a996-fc1e-4cd2-aa6c-74d979fcfaf6)
